ext/featurefmt/dpkg/testdata/valid

```diff
@@ -0,0 +1,2110 @@
+Package: fdisk
```

Comment:
This is directly copied from ubuntu:latest image from docker hub (image id=16508e5c265d)